packages/types/src/session.ts

```diff
@@ -130,11 +130,29 @@ export interface SessionResource extends ClerkResource {
   __internal_toSnapshot: () => SessionJSONSnapshot;
 }
 
+/**
+ * Represents a session resource that has completed all pending tasks
```

Comment:
It might be too soon to document the concept of tasks!

Reply:
> That might be worth the tradeoff of making sure we don't forget to add this later.

I went with the same mindset when making this change! It's difficult to find a good balance here cause pending is already related with the concept of tasks, so I wanted to clarify what it means and it's difference with active 

I think it's quite rate when developers directly import this type definition, so it's mostly for our own internal DX purposes